src/components/progress-bar/progress-bar.less

```diff
@@ -5,30 +5,40 @@
   border-radius: 5px;
   height: 10px;
   background-color: #e7eeec;
+  overflow: hidden;
 
   &-chart-wrapper {
     padding-top: 20px;
     margin-bottom: 0px;
   }
 
   &-bar {
+    position: absolute;
     height: 10px;
-    transition: width 0.2s ease;
+    transition: width 0.2s linear, background-color 0.2s ease;
```

Comment:
`ease` on the `width` gives a weird jumpiness when bar progresses, `linear` gives a smoother movement feeling